src/NHibernate.Test/Generatedkeys/Identity/IdentityGeneratedKeysTest.cs

```diff
@@ -43,7 +58,7 @@ public void IdentityColumnGeneratedIds()
 			}
 		}
 
-		[Test, Ignore("Not supported yet.")]
+		[Test]
 		public void PersistOutsideTransaction()
```

Comment:
These tests have this assert: `Assert.IsNotNull(id, "identity column did not force immediate insert");` which is not correct, because id is long which is never null.